### PR TITLE
add overwrite prompt

### PR DESCRIPTION
**Pull request by kazaamjt**

### CODE REVIEW

File: src/inmanta/main.py [hunk 1]
```diff
@@ -294,8 +294,13 @@ def save_config(client: Client, env: Dict[str, str]):
         "host": client.host,
         "port": client.port,
     }
+
     if os.path.exists(".inmanta"):
```
Comment (bartv) [inline src/inmanta/main.py]:
```suggestion
    if os.path.exists(".inmanta") and not click.confirm(".inmanta exits, do you want to overwrite it?"):
```

Comment (bartv) [reply src/inmanta/main.py]: Then the double write is not required.

File: src/inmanta/main.py [hunk 2]
```diff
@@ -294,8 +294,13 @@ def save_config(client: Client, env: Dict[str, str]):
...
-        click.echo(".inmanta exists, not writing config", err=True)
+        if click.confirm(".inmanta exits, do you want to overwrite it?"):
```
Comment (arnaudsjs) [inline src/inmanta/main.py]:
```suggestion
        if click.confirm(".inmanta exists, do you want to overwrite it?"):
```

File: src/inmanta/main.py [hunk 3]
```diff
@@ -294,8 +294,13 @@ def save_config(client: Client, env: Dict[str, str]):
         "host": client.host,
         "port": client.port,
     }
-    if os.path.exists(".inmanta"):
-        click.echo(".inmanta exists, not writing config", err=True)
+
+    if os.path.exists(".inmanta") and not click.confirm(".inmanta exits, do you want to overwrite it?"):
```
Comment (arnaudsjs) [inline src/inmanta/main.py]: This code looks really strange. It will ask for a confirmation twice. Let's add a test case.

Comment (kazaamjt) [reply src/inmanta/main.py]: resolved. I added a test and cleaned up up the rest.

**Pull request merged**